scipy/stats/_axis_nan_policy.py

```diff
@@ -362,6 +393,12 @@ def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
             # Consolidate other positional and keyword args into `kwds`
             kwds.update(d_args)
 
+            keepdims = kwds.get("keepdims")
+            if keepdims is not None:
+                # remove keepdims to avoid passing it to the underlying
+                # hypotest function
+                kwds.pop("keepdims")
```

Comment:
```suggestion
                keepdims = kwds.pop("keepdims", False)
```
Doesn't this work?
Also, I think this belongs below, where we do `kwds.pop('axis'...)` and `kwds.pop('nan_policy'...)`.